src/qt/transactionview.cpp

```diff
@@ -218,6 +218,7 @@ void TransactionView::setModel(WalletModel *_model)
     {
         transactionProxyModel = new TransactionFilterProxy(this);
         transactionProxyModel->setSourceModel(_model->getTransactionTableModel());
+        transactionProxyModel->setSortRole(Qt::EditRole);
```

Comment:
This line is a duplicate of that one:
https://github.com/bitcoin/bitcoin/blob/9096276e0b2d5b7e19af9a5f3c144ef108ee55e0/src/qt/transactionview.cpp#L225

Reply:
ccv

Reply:
re: https://github.com/bitcoin/bitcoin/pull/15203#discussion_r249168229

> This line is a duplicate of that one:

Good catch. I think it means this line is not actually doing anything and can be removed.

Reply:
@ryanofsky why does the default `Qt::DisplayRole` does not work in this case?

Reply:
> but it looks like the intention was to use EditRole to sort based on unformatted values, while letting DisplayRole return formatted values:

Yep.

Reply:
@ryanofsky @laanwj thanks for clarification.